plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -281,27 +353,39 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 		p.graph, err = graphviz.GenGraph(p.lastTf)
 		if err != nil {
 			log.Printf("Failed to generate traceflow graph \"%s\", err: %s", tfName, err)
+			alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Failed to generate traceflow graph, "+
+				"err: %s", err), action.DefaultAlertExpiration)
+			request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
 			return nil
 		}
 		return nil
 	case showGraphAction:
 		name, err := request.Payload.String(traceNameCol)
 		if err != nil {
 			log.Printf("Failed to get name at string: %s", err)
+			alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Duplicate traceflow: same source pod "+
```

Comment:
Why do we show less than one second when it fails to parse the string?

Reply:
Updated the log.